scripts-python/create_test

```diff
@@ -48,7 +48,7 @@ OR
 
```

Comment:
I don't think that line 27 is quite correct:  there are three new flags --xml_category, --xml_machine --xml_compiler, if any one or more of them are provided then we do not expect a <TEST|SUITE> argument.  How do I express that here?